lib/localhost/authority.rb

```diff
@@ -49,6 +49,7 @@ def self.fetch(*arguments, **options)
 		# @parameter root [String] The root path for loading and saving the certificate.
 		def initialize(hostname = "localhost", root: self.class.path)
 			@root = root
+			@old_root = File.expand_path("~/.localhost")
```

Comment:
I think we can just put this into a constant or compute it in `dir_migrate`?

Reply:
Moved it into migration method.